File: wlr/OGUR/OGUR/GameObjects/Wall.cs
using OGUR.Collision;
using OGUR.Sprites;

namespace OGUR.GameObjects
{
    public class Wall : Entity
    {
        public Wall(Point2 location)
        {
            Initialize(location, SpriteType.WALL, GameObjectType.WALL,Depth.Wall);
            m_isBlocking = true;
        }
    }
}

File: code/aigilas-ported/src/com/aigilas/entities/Wall.java
package com.aigilas.entities;import com.xna.wrapper.*;import java.util.*;import com.spx.core.*;import com.aigilas.management.*;import com.spx.entities.*;
    public class Wall  extends  Entity
    {
        public Wall(Point2 location)
        {
            Initialize(location, SpriteType.WALL, com.aigilas.EntityType.WALL,com.aigilas.Depth.Wall);
            _isBlocking = true;
        }
    }
